tests/test_mongo.py

```diff
@@ -103,18 +104,28 @@ def setUp(self):
         self._remove()
         self.connector = Connector(
             mongo_address=self.repl_set.uri,
-            ns_set=['test.test'],
+            ns_set=['test.test', 'rename.me'],
+            dest_mapping={'rename.me': 'new.target',
+                          'rename.me2': 'new2.target2'},
             doc_managers=(self.mongo_doc,),
             gridfs_set=['test.test'],
             **connector_opts
         )
 
-        self.conn.drop_database('test')
+        self.drop_all_databases()
 
         self.connector.start()
         assert_soon(lambda: len(self.connector.shard_set) > 0)
         assert_soon(lambda: sum(1 for _ in self._search()) == 0)
 
+    def drop_all_databases(self):
+        for name in self.mongo_conn.database_names():
+            if name != "local":
```

Comment:
Maybe add the 'admin' database here as well, since that would allow us to test with auth enabled. We haven't done that in awhile, but it should be possible.